### PR TITLE
[sql/en] More examples

### PR DESCRIPTION
**Pull request by Th3G33k**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

Add more inline code examples, that might be useful :
- multiline comments /* */
- ORDER BY ASC / DESC
- AS
- IN, NOT IN, BETWEEN, AND, OR
- GROUP BY, HAVING
- UNION

### CODE REVIEW

File: sql.html.markdown [hunk 1]
```diff
@@ -63,6 +84,24 @@ SELECT COUNT(*) FROM departments;
 -- have 'en' as a substring of the dept_name value.
 SELECT COUNT(*) FROM departments WHERE dept_name LIKE '%en%';
 
+-- Aggregate functions can be used, with GROUP BY, to compute a value
+-- from a set of values. Most commonly used functions are :
```
Comment (vendethiel) [inline sql.html.markdown:88]: no space before `:`

File: sql.html.markdown [hunk 2]
```diff
@@ -63,6 +84,24 @@ SELECT COUNT(*) FROM departments;
...
+-- MIN(), MAX(), COUNT(), SUM(), AVG().
+-- Use HAVING to filter rows using aggregated values.
```
Comment (vendethiel) [inline sql.html.markdown:90]: using => by

File: sql.html.markdown [hunk 3]
```diff
@@ -75,6 +114,15 @@ SELECT employees.first_name, employees.last_name,
 FROM titles INNER JOIN employees ON
        employees.emp_no = titles.emp_no LIMIT 10;
 
+-- Combine the result of multiple SELECT.
+-- UNION selects distinct rows, UNION ALL selects all rows.
+SELECT * FROM departments WHERE dept_no = 'd001'
+UNION
+SELECT * FROM departments WHERE dept_no = 'd002'
+
+-- SQL syntax order is :
```
Comment (vendethiel) [inline sql.html.markdown:123]: no space before `:`

**Review by vendethiel — CHANGES_REQUESTED** [review]

File: sql.html.markdown [hunk 4]
```diff
@@ -63,6 +84,24 @@ SELECT COUNT(*) FROM departments;
...
+-- from a set of values. Most commonly used functions are:
+-- MIN(), MAX(), COUNT(), SUM(), AVG().
+-- Use HAVING to filter rows by aggregated values.
+
+-- Retrieve the total number of employees, by department number,
+-- with the condition of having more than 100 employees.
+SELECT dept_no, COUNT(dept_no) FROM dept_emp GROUP BY dept_no
+HAVING COUNT(dept_no) > 100
+
+-- Aliases, using the optional keyword AS, can be used for column / table names.
```
Comment (verhovsky) [inline sql.html.markdown:97]:
```suggestion
-- Aliases, using the optional keyword AS, can be used for column/table names.
```

**Comment by Th3G33k** [comment]
Thanks @vendethiel for the review. I applied the suggestions

**Review by vendethiel — APPROVED** [review]

File: sql.html.markdown [hunk 5]
```diff
@@ -3,6 +3,8 @@ language: SQL
 filename: learnsql.sql
 contributors:
   - ["Bob DuCharme", "http://bobdc.com/"]
+  - ["Th3G33k", "https://github.com/Th3G33k"]
+
```
Comment (verhovsky) [inline sql.html.markdown:7]:
```suggestion
```

**Pull request merged**